pysquared/beacon.py

```diff
@@ -19,9 +19,9 @@
 from collections import OrderedDict
 
 try:
-    from mocks.circuitpython.microcontroller import Processor
-except ImportError:
     from microcontroller import Processor
+except ImportError:
+    from mocks.circuitpython.microcontroller import Processor
```

Comment:
Not related to this PR: switching this order is possible now because of #304. With this order change, the IDE shows more accurate hints.